mne/channels/tests/test_channels.py

```diff
@@ -366,6 +366,31 @@ def test_drop_channels():
     pytest.raises(ValueError, raw.drop_channels, 5)  # must be list or str
 
 
+def test_add_reference_channels():
+    """ Test if there is a new reference channel that consist of all zeros"""
+    raw = read_raw_fif(raw_fname, preload=True)
+    raw_original_channels_number = len(raw.ch_names)
```

Comment:
```suggestion
    n_raw_original_channels = len(raw.ch_names)
```
I find it more explicit. A var that starts with n_ is a count (int). A var that starts with raw_ suggests it's a raw instance.